content/en/platform/corda/5.1/deploying-operating/observability/metrics/task-manager.md

```diff
@@ -0,0 +1,55 @@
+---
+date: '2023-10-26'
+version: 'Corda 5.1'
+title: "Task Manager"
+menu:
+  corda51:
+    parent: corda51-cluster-metrics
+    identifier: corda51-cluster-task-manager
+    weight: 1450
+section_menu: corda51
+---
+
+# Task Manager
+
+The task manager exposes metrics for underlying thread pools and jobs processing duration and count.
+
+<style>
+table th:first-of-type {
+    width: 25%;
+}
+table th:nth-of-type(2) {
+    width: 10%;
+}
+table th:nth-of-type(3) {
+    width: 20%;
+}
+table th:nth-of-type(4) {
+    width: 45%;
+}
+</style>
+
+| Metric                                                    | Type    | Tags                                                      | Description                                                                            |
+| :-------------------------------------------------------- | :------ | :-------------------------------------------------------- | :------------------------------------------------------------------------------------- |
+| `corda_taskmanager_executor_active_threads`               | Gauge   | `name`                                                    | The approximate number of threads that are actively executing tasks.                   |
+| `corda_taskmanager_executor_completed_tasks_total`        | Counter | `name`                                                    | The approximate number of tasks that have completed execution.                         |
+| `corda_taskmanager_executor_idle_seconds_count`           | Counter | `name`                                                    | The number of events that have been observed for the base metric.                   |
+| `corda_taskmanager_executor_idle_seconds_max`             | Gauge   | `name`                                                    |                                                                                        |
```

Comment:
@lankydan do we have a description for this one?

Reply:
"The max observed value for the base metric" I guess.

After writing that it doesn't feel the most useful, a lot of the descriptions here are the same (they're just taken from the information that Grafana displays for these metrics).

We can go through and give each of them better descriptions but it will take more time as each one is unique.

Reply:
Have raised https://r3-cev.atlassian.net/browse/DOC-6080